tests/integration/test_remoted/test_socket_communication/test_syslog_message_parser.py

```diff
@@ -0,0 +1,150 @@
+                         ids=t1_cases_ids)
+def test_syslog_message_ipv6(configuration, metadata, set_wazuh_configuration, truncate_event_logs,
+                             restart_wazuh_daemon_function):
+    '''
+    description: Check if 'wazuh-remoted' can receive syslog messages through the socket.
+
+    test_phases:
+        - setup:
+            - Apply ossec.conf configuration changes according to the configuration template and use case.
+            - Truncate wazuh event logs.
+            - Restart wazuh-manager service to apply configuration changes.
+        - test:
+            - Check that the messages are parsed correctly in the alerts.json file.
+        - tierdown:
+            - Truncate wazuh logs.
+            - Restore initial configuration, both ossec.conf and local_internal_options.conf.
+
+    wazuh_min_version: 4.4.0
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Get configurations from the module.
+        - metadata:
+            type: dict
+            brief: Get metadata from the module.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Apply changes to the ossec.conf configuration.
+        - truncate_event_logs:
+            type: fixture
+            brief: Truncate wazuh event logs.
+        - restart_wazuh_daemon_function:
+            type: fixture
+            brief: Restart the wazuh service.
+
+    assertions:
+        - Verify the syslog message is received and parsed correctly.
+
+    input_description:
+        - The `configuration_syslog_message_parser` file provides the module configuration for this
+          test.
+        - The `cases_syslog_message_parser` file provides the test cases.
+
+    expected_output:
+        - fr'"full_log":"{message}".*"location":"{location}"'
+    '''
+    # Set syslog simulator parameters according to the use case data
+    syslog_simulator_parameters = {'address': metadata['address'], 'port': metadata['port'],
+                                   'protocol': metadata['protocol'],
+                                   'messages_number': metadata['messages_number'],
+                                   'message': metadata['message']}
+
+    # Run syslog simulator thread
+    syslog_simulator_thread = ThreadExecutor(syslog_simulator, {'parameters': syslog_simulator_parameters})
+    syslog_simulator_thread.start()
+
+    # Wait until syslog simulator is started
+    sleep(SYSLOG_SIMULATOR_START_TIME)
+
+    # Read the events log data
+    events_data = file.read_file(ARCHIVES_JSON_PATH).split('\n')
+
+    message = metadata['message']
+    location = metadata['address']
+    find_msg = (fr'"full_log":"{message}".*"location":"{location}"').replace('"', r'\"')
+
+    event_msg = [event for event in events_data if bool(re.match(fr".*{find_msg}.*", event))]
+
+    assert len(event_msg) == metadata['messages_number'], f"The events were not parsed as: {find_msg}"
```

Comment:
```suggestion
    assert len(event_msg) == metadata['messages_number'], 'The event's format is not the expected one'
```

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3609/commits/b6c0aac937531f132aeb56fd1e5f4d49f589fd39)